internal/ingest/ingest.go

```diff
@@ -113,6 +126,16 @@ type IngestStats struct {
 	Errors              []string
 }
 
+// Ingest retrieves whitelisted entities from one charmstore and adds them to another, returning statistics on
```

Comment:
We don't generally care that much about line length, but it's perhaps worth keeping comment
width reasonable - I use a little shell script called cfmt that runs the comment through fmt(1).

	#!/usr/bin/env rc
	9 sed 's:^(	*)//:\1:' | 9 fmt | 9 sed 's:^(	*):\1//:'